src/main/java/org/jabref/gui/IconTheme.java

```diff
@@ -261,13 +264,13 @@ public static URL getIconUrl(String name) {
         FILE_POWERPOINT(MaterialDesignIcon.FILE_POWERPOINT), /*css: file-powerpoint */
         FILE_TEXT(MaterialDesignIcon.FILE_DOCUMENT), /*css: file-document */
         FILE_MULTIPLE(MaterialDesignIcon.FILE_MULTIPLE), /*css: file-multiple */
-        FILE_OPENOFFICE(IconTheme.getImage("openoffice")),
-        APPLICATION_EMACS(IconTheme.getImage("emacs")),
-        APPLICATION_LYX(IconTheme.getImage("lyx")),
-        APPLICATION_TEXSTUDIO(IconTheme.getImage("texstudio")),
-        APPLICATION_TEXMAKER(IconTheme.getImage("texmaker")),
-        APPLICATION_VIM(IconTheme.getImage("vim")),
-        APPLICATION_WINEDT(IconTheme.getImage("winedt")),
+        FILE_OPENOFFICE(JabRefMaterialDesignIcon.OPEN_OFFICE),
+        APPLICATION_EMACS(JabRefMaterialDesignIcon.EMACS),
+        APPLICATION_LYX(JabRefMaterialDesignIcon.LYX),
```

Comment:
Are there other places that still use `IconTheme.getImage`? If not this method can be removed as well as `InternalFileIcon`.

Reply:
Refactored this out. I initially left it because maybe someone still needs a pixel icon in future. But since it is really easy now to add SVG icons if we indeed don't find anything appropriate in the existing ones, this can be removed.